circleguard/settings.py

```diff
@@ -204,7 +207,7 @@ def on_setting_changed_from_gui(self, value):
         "message_loading_replays"
     ],
     "1.3.0": [
-        "cache_location",
+        "cache_dir",
```

Comment:
are we putting newly created settings in here? No, right, just changed ones? `cache_dir` is technically a new setting. I think we should put new settings here as well to avoid having a setting, changing the name, then resuing the old name for a new setting some point in the future. Probably won't ever happen but I think it's better to cover our bases and put any new or changed setting here (in which case `visualizer_frametime` and `render_beatmap` need to go here too)